folksonomy/api.py

```diff
@@ -957,11 +957,14 @@ async def check_property_clash(
                 "new_value": conflict[2],
             })
     
-    return PropertyClashCheck(
-        products_with_both=len(conflicting_products),
-        products_with_old_only=old_only_count,
-        products_with_new_only=new_only_count,
-        conflicting_products=conflicts,
+    return JSONResponse(
+        status_code=200,
+        content={
+            "products_with_both": len(conflicting_products),
+            "products_with_old_only": old_only_count,
+            "products_with_new_only": new_only_count,
+            "conflicting_products": conflicts,
+        },
```

Comment:
nitpik:
I liked the fact that you used PropertyClashCheck before,

Isn't it working with:
```suggestion
    return JSONResponse(
        status_code=200,
        content=PropertyClashCheck(
            products_with_both=len(conflicting_products),
            products_with_old_only=old_only_count,
            products_with_new_only=new_only_count,
            conflicting_products=conflicts,
        ),
```


Or maybe:
```suggestion
    return JSONResponse(
        status_code=200,
        content=PropertyClashCheck(
            products_with_both=len(conflicting_products),
            products_with_old_only=old_only_count,
            products_with_new_only=new_only_count,
            conflicting_products=conflicts,
        ).model_dump_json(),
```

Reply:
resolved